etna/core/mixins.py

```diff
@@ -89,6 +90,44 @@ def to_dict(self):
         params["_target_"] = BaseMixin._get_target_from_class(self)
         return params
 
+    def set_params(self, **params: dict) -> "BaseMixin":
+        """Return estimator instance with modified parameters.
```

Comment:
I think it is better to write:
`Return new object instance with modified parameters`